packages/service-library/tests/rabbitmq/test_rabbitmq_rpc.py

```diff
@@ -350,11 +350,12 @@ async def _a_handler() -> None:
         pass
 
     if expect_fail:
-        with pytest.raises(ValidationError) as exec_info:
+        with pytest.raises(
+            ValidationError, match=r"String should have at most \d+ characters"
```

Comment:
Please put the 255 back. This is hanging the behaviour of the test. If someone accidentally changes this limit the test will no longer fail.